src/middleware/render-voyager-page.ts

```diff
@@ -49,8 +49,7 @@ export default function renderVoyagerPage(options: MiddlewareOptions) {
             'Accept': 'application/json',
             'Content-Type': 'application/json',
           }, ${headersJS}),
-          body: JSON.stringify({query: introspectionQuery }),
-          credentials: 'include',
```

Comment:
What's the problem with `credentials`?

Reply:
https://github.com/APIs-guru/graphql-voyager/issues/127